testsuite/tests/translprim/comparison_table.ml.reference

```diff
@@ -232,7 +232,7 @@
                 (apply f (field 0 param) (field 1 param)))
             map =
               (function f l
-                (apply (field 12 (global List!)) (apply uncurry f)
+                (apply (field 14 (global List!)) (apply uncurry f)
```

Comment:
I think this test exposes the offset of some functions (`List.map` here, I suppose) in the `List` module, so it may change when new definitions are added before the functions used.